src/StmtToHTML.cpp

```diff
@@ -1070,11 +1080,17 @@ class HTMLCodePrinter : public IRVisitor {
         context_stack_tags.push_back(tag);
     }
 
-    void print_opening_tag(const std::string &tag, const std::string &cls, std::string id) {
-        stream << "<" << tag << " class='" << cls << "' id='" << id << "'>";
+    /*
```

Comment:
Please delete commented-out code, or add a comment that explains why it is preserved.